### DISCUSSION TITLE
Organizational to-do's

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#2] [Ideas]
This thread can be used by @peterkraus, @davidelbert and myself to organize stuff for the group -- we could consider migrating this to a Project board at some point, but it requires permissions at the GitHub organization level.

- [x] Send out first email to initial proposers
- [x] Organize date for first meeting in mid/late November
- [x] Advertise kick-off event more broadly (MaRDA, RDA, institutional channels)
- [x] Meeting between co-leads to discuss and prepare presentation for kick-off meeting of the initial proposal and work plan (ideally 1 week before the first meeting)
- [x] Begin reaching out to specific projects (from e.g. #3) and people/groups (see comment below)

**Comment by ml-evs** [comment]
More generic projects/bodies to reach out to (for @ml-evs):

- [ ] IUPAC FairSpec
- [x] BIG-MAP data working group
- [x] SINTEF (via OTE-API, listed in #3 already)
- [ ] Henry Royce Institute
- [ ] Physical Sciences Data Infrastructure (UK)

**Comment by ml-evs** [comment]
Quick first draft of possible text for general advert (will return to this later today).


> We are pleased to announce the formation of a new MaRDA working group, "Metadata extractor interoperability in materials science and chemistry", led by Matthew Evans (UCLouvain), Peter Kraus (TU Berlin) and David Elbert (Johns Hopkins University).
> 
> If you are a researcher, scientist and/or developer who transforms raw data between, for example, instrument file formats to an internal data model, or streaming data into databases, or if you write scripts or software libraries that parse or transform data from formats to models, then we would love for you to get involved! 
> 
> Over the next 12 months, we will work to design and implement the following:
> 
> 1. A lightweight metadata schema for parsers (describing inputs and outputs)
> 2. A common API specification for executing parser code (multilingual wrappers for execution via containers, the web or the cloud)
> 3. A searchable registry of parsers (to save grad student time, aid coalescence of communities around existing workhorse parsers, and perhaps render them machine-actionable)
> 
> With these three pillars in place, we envision the following use cases:
> 
> 1. A data infrastructure developer (e.g., institutional repositories, archival services, even journals) can provide rich support for a file format by querying the registry and automatically extracting data locked within uploaded files, thus enabling search and indexing over the contents.
> 2. Ecosystems can be fostered around parser code to improve quality and define more rigorous data models/transformations, thus reducing duplication of efforts.
> 3. Data "providers" (e.g., instrument manufacturers) can be incentivized to plug their own code into the ecosystem and provide enhanced support.
> 
> Our first meeting giving an overview of the proposal, and to begin to design and organize this work, will be on [November 30th 2022 at 14:00 UTC/15:00 CET/09:00 EST](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221130T14).
> 
> There are many ways to potentially contribute, from attending discussion meetings, leaving feedback asynchronously on GitHub, new code development as well as testing and integration with existing libraries.
> 
> To learn more, please visit the GitHub repository [marda-alliance/metadata_extractors](https://github.com/marda-alliance/metadata_extractors) and introduce yourself in the ongoing [discussions](https://github.com/marda-alliance/metadata_extractors/discussions).

**Reply by PeterKraus** [reply]
> We are pleased to announce the formation of a new MaRDA working group, "Metadata extractor interoperability in materials science and chemistry", lead by Matthew Evans (UCLouvain), Peter Kraus (TU Berlin) and David Elbert (Johns Hopkins University).

lead -> led

> A data infrastructure developer (e.g., institutional repositories, archival services, even journals) can provide rich support for a file format by querying the registry and automatically extracting data locked within uploaded files thus enabling search and indexing over the contents.

comma before "thus"

Other than that, great!
